src/com/theironyard/novauc/ATMProgram.java

```diff
@@ -0,0 +1,131 @@
+        balance = (int) (Math.random() * 5000);
+    }
+
+    public void setChange() {
+        quarters = (withdraw / .25);
+        remainder = (withdraw % .25);
+        dimes = (remainder / .10);
+        remainder = (remainder % .10);
+        nickles = (remainder / .05);
+        remainder = (remainder % .05);
+        pennies = (remainder / .01);
+
+        squarters = (int) quarters;
+        sdimes = (int) dimes;
+        snickles = (int) nickles;
+        spennies = (int) pennies;
+    }
+
+    public void checkBalance() throws Exception {
+        System.out.println("#################  BALANCE MENU #################");
+        System.out.println();
+        System.out.println("Your current Balance is    " + balance);
+        System.out.println();
+        System.out.println("would you like to return to the main menu: Y/N ");
+        Main.kb.nextLine();
+        String options = Main.kb.nextLine();
+
+        if (options.equalsIgnoreCase("y")) {
+            Program();
+        }
+        else if (options.equalsIgnoreCase("n")) {
+            cancel();
+        }
+        else
+            throw new Exception("you have chosen an invalid option");
+    }
+
+
+
+    public void withdrawFunds()throws Exception{
+        System.out.println("################  WITHDRAW MENU #################");
+        System.out.println();
+        System.out.println("PLEASE ENTER THE AMOUNT YOU WISH TO WITHDRAW");
+        withdraw=Main.kb.nextDouble();
+
+        if (withdraw>balance){
+            throw new Exception("You have entered a larger amount than what is available");
+        }
+        else {
+            System.out.println("CURRENT BALANCE:    " + balance);
+            System.out.println("\tMINUS");
+            System.out.println("WITHDRAWN AMOUNT:   " + withdraw);
+            System.out.println("WOULD YOU LIKE TO CONTINUE WITH THE WITHDRAW: Y/N");
+            System.out.println();
+            Main.kb.nextLine();
+            String response = Main.kb.nextLine();
+            if (response.equalsIgnoreCase("n")) {
+                Program();
+            }
+            else if (response.equalsIgnoreCase("y")){
+                setChange();
+                System.out.println("YOU WILL BE RECEIVING");
+                System.out.println(squarters+" QUARTERS "+sdimes+" DIMES "+snickles+" NICKLES "+spennies+" PENNIES ");
+                System.out.println("TRANSACTION HAS BEEN COMPLETED PLEASE TAKE YOUR MONEY");
+                balance = balance - withdraw;
+                System.out.println();
+                System.out.println("#################################################");
+                System.out.println();
+                System.out.println("CURRENT AMOUNT IN ACCOUNT IS NOW " + balance);
+                System.out.println();
+                System.out.println("would you like to return to the main menu: Y/N ");
+                Main.kb.nextLine();
+                String options = Main.kb.nextLine();
+
```

Comment:
Incredible job.  Ludicrous mode is awesome.  One minor bug here on line 84 and 85 you call kb.nextLine twice so you have to input y twice to more forward, but this is really minor.  Again really impressive work.